src/main/java/org/broadinstitute/consent/http/resources/DataRequestCasesResource.java

```diff
@@ -60,13 +60,4 @@ public Response getMatchSummaryCases(@Auth AuthUser authUser) {
     }
 
 
-    @GET
-    @Path("/closed")
-    @Produces("application/json")
-    @RolesAllowed({CHAIRPERSON, MEMBER, ALUMNI, ADMIN})
-    public Response describeClosedElections(@Auth AuthUser authUser) {
-        List<Election> elections = electionService.describeClosedElectionsByType(ElectionType.DATA_ACCESS.getValue(), authUser);
-        return Response.ok().entity(elections).build();
-    }
-
```

Comment:
There are a few unused imports here as a result of this. Also, `electionService` is now unused and we can remove a good chunk of plumbing from that.